src/Microsoft.ComponentDetection.Detectors/pip/PythonResolver.cs

```diff
@@ -210,10 +214,57 @@ private void AddGraphNode(PythonResolverState state, PipGraphNode parent, string
         }
         else
         {
-            var node = new PipGraphNode(new PipComponent(name, version));
+            var node = new PipGraphNode(new PipComponent(name, version, license: license, author: author));
             state.NodeReferences[name] = node;
             parent.Children.Add(node);
             node.Parents.Add(parent);
         }
     }
+
+    private string GetAuthorFromProject(PythonProject project)
+    {
+        if (!string.IsNullOrEmpty(project.Info?.Author))
+        {
+            return project.Info.Author;
+        }
+
+        if (!string.IsNullOrEmpty(project.Info?.Maintainer))
+        {
+            return project.Info.Maintainer;
+        }
+
+        if (!string.IsNullOrEmpty(project.Info?.AuthorEmail))
+        {
+            return project.Info.AuthorEmail;
+        }
+
+        if (!string.IsNullOrEmpty(project.Info?.MaintainerEmail))
+        {
+            return project.Info.MaintainerEmail;
+        }
```

Comment:
Why are we selecting this priority for project author? Is it documented anywhere? Some doc links in PR description would be helpful to get more context even for future reviewers.

Reply:
I moved these around a bit after reading this comment. The order I chose was really to make the field more representative of who is responsible for the contents inside of the package. The author may no longer be involved with a project so prioritizing the maintainer felt appropriate in this case. But there is no spec on this. Just what I feel would make sense if the field is going to be a part of an SBOM.

Reply:
I think the new order makes sense - if we need to update it in the future we can do so.